packages/controller/src/resources/monitoring/system/dashboards/controller-manager.json

```
@@ -504,7 +504,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "histogram_quantile(0.99, sum(rate(rest_client_request_latency_seconds_bucket{job=\"kube-controller-manager\", instance=~\"$instance\", verb=\"POST\"}[5m])) by (verb, url, le))",
+              "expr": "histogram_quantile(0.99, sum(rate(rest_client_request_latency_seconds_bucket{job=\"apiserver\", instance=~\"$instance\", verb=\"POST\"}[5m])) by (verb, url, le))",
```

The graph is empty and it seems the metric is deprecated https://docs.signalfx.com/en/latest/integrations/agent/monitors/kubernetes-scheduler.html

```suggestion
              "expr": "histogram_quantile(0.99, sum(rate(rest_client_request_duration_seconds_bucket{job=\"apiserver\", instance=~\"$instance\", verb=\"POST\"}[5m])) by (verb, url, le))",
```